applications/openshift/ocp-permissions/ocp-files/file_permissions_openshift_node_client_crt/rule.yml

```diff
@@ -30,4 +30,4 @@ template:
     name: file_permissions
     vars:
         filepath: /etc/origin/node/client-ca.crt
-        filemode: '0600'
+        filemode: '0644'
```

Comment:
@rmetrich need a good reason (other than this is what OCP does) for changing file permissions from a good restricted setting to a more open restricted setting. Thanks!

Reply:
@redhatrises Sure, the reason for loosing permissions to 0644 is to stick to the rule description itself:

~~~
description: |-
    {{{ describe_file_permissions(file="/etc/origin/openvswitch/*", perms="0644") }}}
~~~

So either the description should be changed, or filemode loosen.

Reply:
Great catch! Thanks.